packages/react-components/src/queries/useHistoricalAssetPropertyValues/requestExecution/getHistoricalAssetPropertyValueRequest.ts

```diff
@@ -1,33 +1,91 @@
 import {
   GetAssetPropertyValueHistory,
-  viewportEndDate,
-  viewportStartDate,
+  RequestResponse,
 } from '@iot-app-kit/core';
 import { HistoricalAssetPropertyValueRequest } from '../types';
+import { take } from 'lodash';
+import { mapViewport } from './mapViewport';
+import { mapTimeOrdering } from './mapTimeOrdering';
+
+const DEFAULT_MAX_RESULTS = 20000;
```

Comment:
nit: this is already defined in the batcher file

Reply:
We can make a constants file, but since these are different apis, they could potentially support different max results.